lib/tasks/generateResourcesJson.js

```diff
@@ -0,0 +1,94 @@
+"use strict";
+
+const resourceListCreator = require("../processors/resourceListCreator");
+
+function getCreatorOptions(projectName) {
+	const creatorOptions = {};
+	if ( projectName === "sap.ui.core" ) {
+		Object.assign(creatorOptions, {
+			externalResources: {
+				"sap/ui/core": [
+					"*",
+					"sap/base/",
+					"sap/ui/"
+				]
+			},
+			mergedResourcesFilters: [
+				"jquery-sap*.js",
+				"sap-ui-core*.js",
+				"**/Component-preload.js",
+				"**/library-preload.js",
+				"**/library-preload-dbg.js",
+				"**/library-preload.json",
+				"**/library-all.js",
+				"**/library-all-dbg.js",
+				"**/designtime/library-preload.designtime.js",
+				"**/library-preload.support.js"
+			].join(",")
+		});
+	}
+	return creatorOptions;
+}
+
+/**
+ * Task for creating a resources.json file, describing all productive build resources.
```

Comment:
Should we add a link to the relevant documentation: https://openui5.hana.ondemand.com/#topic/adcbcf8b50924556ab3f321fcd9353ea ?

Reply:
done